core/controllers/profile.py

```diff
@@ -79,6 +79,7 @@ def get(self, username):
 
         self.values.update({
             'user_bio': user_settings.user_bio,
+            'subject_interests': ", ".join(user_settings.subject_interests),
```

Comment:
I'm not sure how to convert a list into a string in html/css. Where/how would I do this?